src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/GlobalSearchService.kt

```diff
@@ -112,7 +112,6 @@ class GlobalSearchService(
 
     val scroll = Scroll(TimeValue.timeValueMinutes(1L))
     val searchResponse = setupIndexSearch(scroll)
-    log.info(searchResponse.toString())
```

Comment:
need to get rid of these now things are working as they dump out the whole index records !